eZ/Bundle/EzPublishCoreBundle/EventListener/BackwardCompatibleCommandListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\EventListener;
+
+use eZ\Bundle\EzPublishCoreBundle\Command\BackwardCompatibleCommand;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+final class BackwardCompatibleCommandListener implements EventSubscriberInterface
```

Comment:
`DeprecatedCommandNameCommandListener` instead?

At the very least, `BackwardCompatibleCommandListener` class name is not really correct. This class does not actually provide backwards compatibility.

Reply:
True, but that way at least it match interface name.